Doc/library/typing.rst

```diff
@@ -2815,6 +2815,56 @@ ABCs for working with IO
    represent the types of I/O streams such as returned by
    :func:`open`.
 
+The following protocols offer a simpler alternative for common use cases. They
+are especially useful for annotating function and method arguments and are
+decorated with :func:`@runtime_checkable <runtime_checkable>`.
+
+.. class:: Reader[T]
+
+   Protocol for reading from a file or other input stream.
+
+   .. method:: read(size=...)
+
+        Read data from the input stream and return it. If ``size`` is
```

Comment:
I think this will render oddly, check the rendered output.

Reply:
It did render fine, but I fixed the overindentation nevertheless.